runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/RequirementCollections.java

```diff
@@ -0,0 +1,199 @@
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.cluescrolls.clues;
+
+import java.util.stream.Stream;
+import static net.runelite.api.ItemID.*;
+import net.runelite.client.game.ItemVariationMapping;
+import net.runelite.client.plugins.cluescrolls.clues.item.AllRequirementsCollection;
+import net.runelite.client.plugins.cluescrolls.clues.item.AnyRequirementCollection;
+import net.runelite.client.plugins.cluescrolls.clues.item.ItemRequirements;
+import static net.runelite.client.plugins.cluescrolls.clues.item.ItemRequirements.all;
+import static net.runelite.client.plugins.cluescrolls.clues.item.ItemRequirements.any;
+import static net.runelite.client.plugins.cluescrolls.clues.item.ItemRequirements.item;
+import static net.runelite.client.plugins.cluescrolls.clues.item.ItemRequirements.range;
+import net.runelite.client.plugins.cluescrolls.clues.item.SingleItemRequirement;
+
+class RequirementCollections
+{
+	static final AnyRequirementCollection ANY_SLAYER_HELMET = any("Any slayer helmet",
+		ItemVariationMapping.getVariations(SLAYER_HELMET).stream()
+			.map(ItemRequirements::item)
+			.toArray(SingleItemRequirement[]::new));
+	static final AnyRequirementCollection ANY_RING_OF_WEALTH = any("Any ring of wealth",
+		ItemVariationMapping.getVariations(RING_OF_WEALTH).stream()
+			.map(ItemRequirements::item)
+			.toArray(SingleItemRequirement[]::new));
+	static final AnyRequirementCollection ANY_PHARAOHS_SCEPTRE = any("Pharaoh's sceptre",
+		ItemVariationMapping.getVariations(PHARAOHS_SCEPTRE).stream()
+			.map(ItemRequirements::item)
+			.toArray(SingleItemRequirement[]::new));
+	static final AnyRequirementCollection ANY_RUNE_HERALDIC_SHIELD = any("Any Rune shield (h)",
+		ItemVariationMapping.getVariations(RUNE_SHIELD_H1).stream()
+			.map(ItemRequirements::item)
+			.toArray(SingleItemRequirement[]::new));
+	static final AnyRequirementCollection ANY_RUNE_HERALDIC_HELM = any("Any Rune helm (h)",
+		ItemVariationMapping.getVariations(RUNE_HELM_H1).stream()
+			.map(ItemRequirements::item)
+			.toArray(SingleItemRequirement[]::new));
+	static final AnyRequirementCollection ANY_TEAM_CAPE = any("Any team cape",
+		Stream.of(
+				ItemVariationMapping.getVariations(TEAM1_CAPE).stream(),
+				Stream.of(TEAM_CAPE_I, TEAM_CAPE_X, TEAM_CAPE_ZERO))
+			.reduce(Stream::concat)
+			.orElseGet(Stream::empty)
+			.map(ItemRequirements::item)
+			.toArray(SingleItemRequirement[]::new));
+	static final AnyRequirementCollection ACTIVE_CRYSTAL_BOW_OR_BOW_OF_FAERDHINEN = any("Crystal Bow or Bow of Faerdhinen",
+		Stream.of(
+				ItemVariationMapping.getVariations(BOW_OF_FAERDHINEN_INACTIVE).stream(),
+				Stream.of(CRYSTAL_BOW, CRYSTAL_BOW_24123))
+			.reduce(Stream::concat)
+			.orElseGet(Stream::empty)
+			.filter(itemId -> itemId != BOW_OF_FAERDHINEN_INACTIVE)
+			.map(ItemRequirements::item)
+			.toArray(SingleItemRequirement[]::new));
```

Comment:
A single indent for the `Stream.of()` arguments is fine, like the code originally was formatted.
```suggestion
		Stream.of(
			ItemVariationMapping.getVariations(BOW_OF_FAERDHINEN_INACTIVE).stream(),
			Stream.of(CRYSTAL_BOW, CRYSTAL_BOW_24123))
			.reduce(Stream::concat)
			.orElseGet(Stream::empty)
			.filter(itemId -> itemId != BOW_OF_FAERDHINEN_INACTIVE)
			.map(ItemRequirements::item)
			.toArray(SingleItemRequirement[]::new));
```